### DISCUSSION TITLE
Plugin Backend UI Deleted ?????

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#12717] [General]
Hi @christianbader @dvesh3 

This link is not working, is deleted, or what any idea??

https://pimcore.com/docs/pimcore/current/Development_Documentation/Extending_Pimcore/Bundle_Developers_Guide/Plugin_Backend_UI.html

Thanks.

**Comment by jdreesen** [comment]
The functionality has been deprecated (but still works until Pimcore 11) and was replaced by native JS events (see #7290, #12566 and  #12633) in Pimcore 10.5, which was released yesterday.

The new documentation page is: [docs/current/.../Event_Listener_UI.html](https://pimcore.com/docs/pimcore/current/Development_Documentation/Extending_Pimcore/Bundle_Developers_Guide/Event_Listener_UI.html)
But you can still view the old one here: [docs/10.4/.../Plugin_Backend_UI.html](https://pimcore.com/docs/pimcore/10.4/Development_Documentation/Extending_Pimcore/Bundle_Developers_Guide/Plugin_Backend_UI.html)

**Reply by fashxp** [reply]
or use https://pimcore.com/docs/pimcore/10.4/Development_Documentation/Extending_Pimcore/Bundle_Developers_Guide/Plugin_Backend_UI.html for the older version. 

Is that page linked somewhere, where we should fix it?

**Reply by ghost** [reply]
@fashxp no this is page is not linked, but it's working Y'day so I have raised it, now it's clear base on @jdreesen replay.

Thanks.
